tests/Mutant/MutantCreatorTest.php

```diff
@@ -42,10 +71,10 @@ public function test_it_uses_avaialable_file_if_hash_is_the_same()
         $coverage->shouldReceive('hasExecutedMethodOnLine')->andReturn(true);
         $coverage->shouldReceive('getAllTestsFor')->andReturn(['test', 'list']);
 
-        $creator = new MutantCreator($tempdir, $differ, $standard);
+        $creator = new MutantCreator($this->directory, $differ, $standard);
         $mutant = $creator->create($mutation, $coverage);
 
-        $this->assertSame($tempdir . '/mutant.hash.infection.php', $mutant->getMutatedFilePath());
+        $this->assertSame($this->directory . '/mutant.hash.infection.php', $mutant->getMutatedFilePath());
```

Comment:
please replace `/mutant.hash.infection.php` with $this->fileName (or self::FILE_NAME)